src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPTextViewConnectionListener.cs

```diff
@@ -63,13 +106,60 @@ public void SubjectBuffersConnected(ITextView textView, ConnectionReason reason,
             }
 
             RazorLSPTextViewFilter.CreateAndRegister(vsTextView);
+
+            // Initialize the user's options and start listening for changes.
+            _razorEditorOptions = _editorOptionsFactory.GetOptions(textView);
+            Assumes.Present(_razorEditorOptions);
+            RazorOptions_OptionChanged(null, null);
+            _razorEditorOptions.OptionChanged += RazorOptions_OptionChanged;
```

Comment:
To avoid leaking you'll need to detatch from this event on Disconnected, also you'll want to not attach multiple times. Aka, if you have the same document open 2x (aka 2 text views) you don't want to be triggering options changed N number of times